### PR TITLE
Language select

### PR DESCRIPTION
**Pull request by bandvov**

### CODE REVIEW

File: src/store/store.js [hunk 1]
```diff
@@ -9,8 +9,11 @@ const configureStore = () => {
     rootReducer,
     compose(
       applyMiddleware(sagaMiddleware),
-      window.__REDUX_DEVTOOLS_EXTENSION__ &&
-        window.__REDUX_DEVTOOLS_EXTENSION__()
+      compose(
+        applyMiddleware(sagaMiddleware),
+        window.__REDUX_DEVTOOLS_EXTENSION__ &&
+          window.__REDUX_DEVTOOLS_EXTENSION__()
+      )
```
Comment (VolodymyrTrach) [inline src/store/store.js:16]: Why did you add a second compose and applyMiddleware(sagaMiddleware),? There is compose at line 10

File: src/containers/language/language.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import { Select, MenuItem } from '@material-ui/core';
+import {
+  setToLocalStorage,
+  getFromLocalStorage
+} from '../../services/localstorage.service';
+import useStyles from './language.styless';
+import { changeLanguage } from '../../redux/language/language.actions';
+import { LANGUAGES_LIST } from '../../configs';
+
+const languageInLocalStorage = JSON.parse(getFromLocalStorage('language')) || 0;
```
Comment (VolodymyrTrach) [inline src/containers/language/language.js:12]: you don`t need to parse it here. Localstorage service already did it for you

File: src/store/store.js (hunk 1 above)
Comment (bandvov) [reply src/store/store.js:16]: fixed

File: src/containers/language/language.js (hunk 2 above)
Comment (bandvov) [reply src/containers/language/language.js:12]: fixed

File: src/index.css [hunk 3]
```diff
@@ -17,3 +16,18 @@ code {
...
+  padding: 0 !important;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.MuiList-padding {
+  padding: 0 !important;
+}
+.MuiMenuItem-root {
```
Comment (VolodymyrTrach) [inline src/index.css:30]: .MuiMenuItem-root is already written at line 20

Comment (bandvov) [reply src/index.css]: fixed

File: src/utils/client.js [hunk 4]
```diff
@@ -1,7 +1,11 @@
 import ApolloClient, { gql } from 'apollo-boost';
 
+export const REACT_APP_API_URL =
```
Comment (Alan18081) [inline src/utils/client.js:3]: Please, move it to separate merge request, I want  to merge it as quickly as possible

File: src/containers/language/language.styless.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
+import { makeStyles } from '@material-ui/core';
```
Comment (Alan18081) [inline src/containers/language/language.styless.js:1]: Please, rename the file to `language.styles.js`

File: src/services/localstorage.service.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    accessToken: null,
+    _id: null,
+    wishlist: null,
+    cart: null,
+    language: 0
+  };
+  localStorage.setItem('horondi', JSON.stringify(horondi));
+};
+
+if (!localStorage.getItem('horondi')) {
```
Comment (Alan18081) [inline src/services/localstorage.service.js:12]: Please, put this code inside `index.js` or `app.js` to ensure that it is easy to find a place where resetting is performed

File: src/containers/language/language.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+const languageInLocalStorage = getFromLocalStorage('language') || 0;
+
+const Language = () => {
+  const styles = useStyles();
+  const dispatch = useDispatch();
+  dispatch(changeLanguage(languageInLocalStorage));
+
+  const handleChange = (e) => {
+    const targetValue = e.target.value;
+    if (targetValue === languageInLocalStorage) {
```
Comment (Alan18081) [inline src/containers/language/language.js:21]: Maybe it should be `!==` to perform this operation only when another language is selected. Do I miss something?

File: src/containers/navbar-left/navbar-left.js [hunk 8]
```diff
@@ -39,7 +39,7 @@ const NavbarLeft = () => {
   return (
     <Toolbar>
       <Typography variant='h6'>
-        <Link to='/' className={styles.logo}>
+        <Link id='logo' to='/' className={styles.logo}>
```
Comment (Alan18081) [inline src/containers/navbar-left/navbar-left.js:42]: Why do we need an id here?

File: src/services/localstorage.service.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
+export const clearLocalStorage = () => {
```
Comment (Alan18081) [inline src/services/localstorage.service.js:1]: Please, rename the file to `local-storage.service.js`

File: src/containers/language/language.styless.js (hunk 5 above)
Comment (bandvov) [reply src/containers/language/language.styless.js:1]: fixed

File: src/services/localstorage.service.js (hunk 9 above)
Comment (bandvov) [reply src/services/localstorage.service.js:1]: fixed

File: src/containers/language/language.js (hunk 7 above)
Comment (bandvov) [reply src/containers/language/language.js:21]: I have simplified the code. this "if" statement is removed

File: src/containers/navbar-left/navbar-left.js (hunk 8 above)
Comment (bandvov) [reply src/containers/navbar-left/navbar-left.js:42]: I wanted to use it for cypress test. it is not needed any more. fixed

File: src/services/localstorage.service.js (hunk 6 above)
Comment (bandvov) [reply src/services/localstorage.service.js:12]: moved code to index.js
